source/taup/taup_setsac.rst

```diff
@@ -0,0 +1,31 @@
+taup_setsac
+===========
+
+:doc:`taup_time` 命令可以计算各震相的理论走时，
+:doc:`taup_setsac` 命令则用于计算理论走时并将走时信息写入到 SAC 文件头段中。
+
+使用 ``taup_setsac --help`` 可查询该命令的完整用法和选项。
+
+其用法相对简单，示例如下::
+
+   $ taup_setsac -mod prem -ph P-0,S-1,PcP-8,ScS-9 -evdpkm *.SAC
+
+-  ``-mod`` 指定使用 PREM 模型
+-  ``-ph P-0,S-1,PcP-8,ScS-9`` 将 P 波到时标记到 SAC 头段 ``T0`` 中，
+   S 到时标记到 ``T1`` 中，PcP 标记到 ``T8``，ScS 标记到 ``T9``。
+-  ``-evdpkm``\ ：由于历史原因，该命令默认 SAC 文件中的震源深度单位为米，
+   而现在的 SAC 文件中的震源深度单位为千米。该选项设置深度的单位为 km
+
+该命令在将震相走时标记到头段变量 ``Tn`` 的同时，还会修改对应的 ``KTn`` 和 ``USERn``，
+其中 ``KTn`` 中保存震相名，``USERn`` 中保存震相的射线参数（单位为 s/radian）。
+
```

Comment:
```suggestion

使用该命令时，SAC 头段变量 ``EVDP`` 和 ``O` 必须已赋值。
若头段变量 ``GCARC`` 或 ``DIST`` 未赋值，但震源和台站位置的头段变量
（``EVLA``、``EVLO``、``STLA``、``STLO``）已赋值，该命令可以自动计算震中距，
与 :doc:`taup_time` 命令类似。
```

Reply:
> 函数位于 SAC 源码 ``src/ucf/distaz.c`` 中。

I don't think TauP has this code.

Reply:
This is SAC's code. I mean when we write evla evlo stla stlo to SAC header, SAC will call this fucntion to update gcarc and dist.

Reply:
Do we have to revise this comment?